### PR TITLE
[Cherry-pick] fix isfinite all any ops

### PR DESCRIPTION
**Pull request by syyxsxx**
### PR types
Bug fixes

### PR changes
OPs

### Describe
1、delete reduce_all reduce_any in math.py  
2、add all and any in math.py
3、fix all and any example code
4、delete isfinite in logic.py

### CODE REVIEW

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -2071,15 +2071,14 @@ def all(x, axis=None, keepdim=False, name=None):
 
             import paddle
             import paddle.fluid as fluid
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: 这一行也需要删除

File: python/paddle/tensor/math.py [hunk 2]
```diff
@@ -2167,15 +2166,14 @@ def any(x, axis=None, keepdim=False, name=None):
...
```
Comment (TCChenlong) [inline python/paddle/tensor/math.py]: 同上

File: python/paddle/tensor/math.py (hunk 1 above)
Comment (syyxsxx) [reply python/paddle/tensor/math.py]: 已经删除

File: python/paddle/tensor/math.py (hunk 2 above)
Comment (syyxsxx) [reply python/paddle/tensor/math.py]: 已经删除

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]

**Pull request merged**